xtask/src/tasks.rs

```diff
@@ -14,13 +19,37 @@ use xmas_elf::{
     ElfFile,
 };
 
-const VID: u16 = 0x1915;
-const PID: u16 = 0x521f;
+use crate::config;
 
-fn main() -> Result<(), anyhow::Error> {
-    let args = env::args().skip(1 /* program name */).collect::<Vec<_>>();
+pub fn change_channel(channel: &str) -> color_eyre::Result<()> {
```

Comment:
general nitpick about the functions in `tasks.rs`, imo it would improve readability significantly to refactor out some steps of the functions into helpers (for example the file conversioins in `dongle_flash()`). this is probably overkill in terms of time/money for helper tools though.